manticore_challenge/win.py

```diff
@@ -18,11 +18,9 @@
 addrs = []
 
 def get_exits():
-    """ Extract exit calls from each check function using r2 """
-    r2 = r2pipe.open('manticore_challenge')
-    r2.cmd('aaa')
-    exits_disasm = r2.cmd('pdf @@ sym.check*~exit')
-    exits = [int(line.split()[2], 16) for line in exits_disasm.split('\n')]
+    """ Extract exit calls from each check function using objdump """
+    exits_disasm = check_output("objdump -d manticore_challenge | grep exit", shell=True)
```

Comment:
Better to change shell=True and use subprocess, but for this case it should be OK.